### ISSUE TITLE
Feature Request: edit traffic patterns, rather than remove and create new

### ISSUE THREAD

**Issue by fourdegrees** [albar965/littlenavmap#1168]
I've been setting up traffic patterns for my local airports based on researching airport information. I like to customize them based on what I find so I can practice the pattern in MSFS. For example, here is the link for Boulder Municipal Airport's (KBDU) official traffic patterns:

https://bouldercolorado.gov/services/airport-traffic-pattern

And here is a screenshot from their PDF:
![image](https://github.com/albar965/littlenavmap/assets/8021366/8168a07f-9ddd-4d86-9e80-e68289bec226)

I was able to recreate the one for RWY 080 in LNV, but it took a bit of trial-and-error because there is no way to edit a traffic pattern once it has been placed. All I can do is Remove and Add new. It would be wonderful if right-clicking the hot button allowed an option to Edit, in addition to Remove. 

![image](https://github.com/albar965/littlenavmap/assets/8021366/23be305b-07f0-4281-b21a-8647d9e938aa)

I'm in the process of setting up custom patterns for a nearby airport, Rocky Mountain Metropolitan Airport (KBJC), and was inspired to make this request, since I'm back in the loop of Adding a pattern, seeing it doesn't match the diagram, Removing it, using new values, and repeating until it's "just right." Being able to instead Edit a pattern that's already on the map would make it much faster because, for example, I'm setting up RWY 30R's right-hand pattern now. If there's an error, I have to right-click, Remove, then right-click Add, and the dialogue defaults back to RWY 12L with a left pattern, so I have to click 30R, change to a right pattern, then adjust the values, and click OK. If I was off, I have to Remove and start over. 

I'm sure you have bigger projects, but I thought it would at least not hurt to ask. I love your program, and thank you so much for making it available for free! It's amazing. Cheers

**Comment by albar965** [comment]
Makes sense and was already requested a few times. I'll see if I can add this to 3.0.9.
A button `Apply` could help to see the changes immediately.
Alex
